examples/mqtt_temperature.eve

```diff
@@ -0,0 +1,40 @@
+
+# MQTT
+## Temperature
+From mqtt://mqtt.bitraf.no see http://iot.bitraf.no
+### Define the sensors
+```eve
+commit @shared
+  [#bitraf #temperature name: "lab" topic: "bitraf/temperature/1"]
+  [#bitraf #temperature name: "office" topic: "bitraf/temperature/2/value"]
+  [#bitraf #temperature name: "outside" topic: "bitraf/temperature/3/value"]
+```
+### Show latest sensor values
+```eve
+search @shared
+  sensor = [#bitraf #temperature name topic]
+search @mqtt
+  message = [#message #incoming topic: topic, payload]
+  sort[value: payload, per: topic] = 1
```

Comment:
Each message (on that topic) contains a payload with a temperature value. The sensors send such messages at a fixed messages intervals (every minute in our case, I think). And since the search is for messages, without a sort/filter I get all these messages, including things which are long in the past.